schemas/opticalData.json

```diff
@@ -5,50 +5,37 @@
   "description": "Optical data sets of building envelope components. It shall be possible to exchange the results of all important optical measurements and simulations. At the same time, this general format is intended to limit the number of possible representations of one data set to make it easy to develop software which interprets the data correctly.",
   "$defs": {
     "data": {
-      "oneOf": [
-        {
-          "title": "Internal data node with leaf children",
-          "description": "Internal node of the data tree whose children are leaf data point nodes. It specifies component characteristics that are identical for child data points.",
-          "type": "object",
-          "properties": {
-            "componentCharacteristics": {
-              "$ref": "#/$defs/componentCharacteristics",
-              "description": "Component characteristics that are identical for all child data points in `dataPoints`. Such characteristics must be specified here and not in child data points in `dataPoints[*]/componentCharacteristics`."
-            },
-            "dataPoints": {
-              "title": "Optical data points",
-              "type": "array",
-              "items": {
-                "$ref": "#/$defs/dataPoint"
+      "title": "Optical data",
+      "description": "Optical `data` as an array of optical `dataPoints` with their characteristics. An additional property could be added for `serverSpecific` data.",
```

Comment:
I'd use `can be added` instead of `could`. And for consistency I'd write "with their `componentCharacteristics`".